### PR TITLE
Rename Remote Environments in Console UI

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- change visible labels in ui
- change pathSegment in the consol-ui
- adjust ui-tests

### CODE REVIEW

File: core/src/app/content/settings/remote-environments/remote-environment-details/remote-environment-details.component.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h2>General Information</h2>
       <div class="sf-panel">
         <div class="sf-panel__head">
           <div class="sf-toolbar">
-              <h2>Environment Bindings</h2>
+              <h2>Application Bindings</h2>
```
Comment (dariadomagala) [inline core/src/app/content/settings/remote-environments/remote-environment-details/remote-environment-details.component.html]: Shouldn't it stay as `Environment Bindings`? It wasn't **Remote** Environment Bindings

File: core/src/app/content/settings/remote-environments/remote-environments.component.ts [hunk 2]
```diff
@@ -20,9 +20,9 @@ import { ActivatedRoute, Router } from '@angular/router';
...
 })
 export class RemoteEnvironmentsComponent extends AbstractKubernetesElementListComponent {
-  title = 'Remote Environments';
-  emptyListText = 'It looks like you don’t have any remote environments yet.';
-  createNewElementText = 'Add Remote Environment';
+  title = 'Applications';
+  emptyListText = 'It looks like you don’t have any Applications yet.';
+  createNewElementText = 'Add Application';
   baseUrl = AppConfig.k8sApiServerUrl_remoteenvs;
   resourceKind = 'RemoteEnvironment';
```
Comment (dariadomagala) [inline core/src/app/content/settings/remote-environments/remote-environments.component.ts]: Please change that to `apps`.

**Comment by pekura** [comment]
Should we update the event-trigger-chooser.component in Lambda (<div class="col-2">Remote Environment</div>) as well or is there an automatix issue for that?

**Comment by kwiatekus** [comment]
> Should we update the event-trigger-chooser.component in Lambda (
> Remote Environment
> ) as well or is there an automatix issue for that?

Automatix will take care

**Pull request merged**